src/main/kotlin/com/dansplugins/factionsystem/command/faction/claim/MfFactionClaimCommand.kt

```diff
@@ -19,11 +19,37 @@ import java.util.logging.Level.SEVERE
 import kotlin.math.floor
 
 class MfFactionClaimCommand(private val plugin: MedievalFactions) : CommandExecutor, TabCompleter {
+    private val factionClaimAutoCommand = MfFactionClaimAutoCommand(plugin)
     private val factionClaimFillCommand = MfFactionClaimFillCommand(plugin)
+    private val factionClaimCheckCommand = MfFactionClaimCheckCommand(plugin)
+
+    private val autoAliases = listOf("auto", plugin.language["CmdFactionClaimAuto"])
+    private val fillAliases = listOf("fill", plugin.language["CmdFactionClaimFill"])
+    private val checkAliases = listOf("check", plugin.language["CmdFactionClaimCheck"])
+    // private val circleAliases = listOf("circle", plugin.language["CmdFactionCheckClaim"])
```

Comment:
Perhaps we should delegate the existing claim logic into the circle command, given that claiming a circle is what it is currently doing.